packages/twenty-front/src/modules/app/effect-components/RtlEffect.tsx

```diff
@@ -0,0 +1,6 @@
+import { useRtl } from '~/hooks/useRtl';
+
+export const RtlEffect = () => {
```

Comment:
this effect can be avoided, see other comments (we should use a synchronous way here)